stardis/radiation_field/opacities/opacities_solvers/voigt.py

```diff
@@ -5,8 +5,9 @@
 
 GPUs_available = cuda.is_available()
 
-if GPUs_available:
-    import cupy as cp
+# Commenting out until cupy is hooked up
+# if GPUs_available:
+#     import cupy as cp
```

Comment:
Is it intentional to comment this out?

Reply:
Yes, to let the non-cuda version run on moria, since we discovered GPUs_available will be true even if we don't use the cuda environment on that machine. It's a little hacky, but until we hook up the GPU to the simulation (and just implement a flag in the config to turn on the GPU architecture or not), it seems like the easiest workaround to me.